modules/iap/charts/iap/templates/static-ingress.yaml

```diff
@@ -15,13 +15,14 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ .Values.iap.jupyterIngress.name }}
+  name: {{ .Values.iap.ingress.name }}
   annotations:
-    kubernetes.io/ingress.global-static-ip-name:  {{ .Values.iap.jupyterIngress.staticIpName }}
-    networking.gke.io/managed-certificates:  {{ .Values.iap.jupyterIngress.managedCertificates }}
+    kubernetes.io/ingress.global-static-ip-name:  {{ .Values.iap.ingress.staticIpName }}
+    networking.gke.io/managed-certificates:  {{ .Values.iap.managedCertificate.name }}
+    kubernetes.io/ingress.class: "gce"
```

Comment:
do we need to mention this annotation? we already mention on service as neg and ingress true..

Reply:
Yes we do, the ingress true is helping to create the gcp resource like NEG. We still need to specify the k8s resource